src/tasks/Crossgen2Tasks/CommonFilePulledFromSdkRepo/BuildErrorException.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
should we add a comment in these `CommonFilePulledFromSdkRepo` files to indicate these are copies and any changes here should be propagated to sdk or vice-versa?

Reply:
I've put a readme.md in the crossgen2tasks directory